public/javascripts/ingredients.js

```diff
@@ -0,0 +1,100 @@
+//FORM
+var $form = $("#ingredients-form");
+
+//if form submit works
+var onSuccessForm = function (data, status) {
+
+  //TODO- fix bug that replaces ingredients list with 2 empty objects
+  $( "#ingredient-list" ).html(data);
```

Comment:
You probably want $("____").append() which adds to the content rather than replace it.